src/api/EscargotPublic.cpp

```diff
@@ -1235,6 +1235,49 @@ void DebuggerOperationsRef::BreakpointOperations::getStackTrace(DebuggerOperatio
     }
 }
 
+void DebuggerOperationsRef::BreakpointOperations::getScopeChain(uint32_t stateIndex, DebuggerOperationsRef::ScopeChainVector& outScopeChain)
+{
+    outScopeChain.clear();
+
+    ExecutionState* state = toImpl(m_executionState);
+
+    while (stateIndex > 0) {
+        state = state->parent();
+        stateIndex--;
+
+        if (state == 0) {
```

Comment:
Would you fix it to compare with `nullptr` (state is a pointer, so its better to compare it with null pointer) ?